helix-core/src/test/java/org/apache/helix/integration/task/TestTaskSchedulingTwoCurrentStates.java

```diff
@@ -138,6 +139,17 @@ public void testTargetedTaskTwoCurrentStates() throws Exception {
     JobQueue.Builder jobQueue = TaskTestUtil.buildJobQueue(jobQueueName);
     jobQueue.enqueueJob("JOB0", jobBuilder0);
 
+    // Make sure master has been correctly switched to Participant1
+    boolean isMasterSwitchedToCorrectInstance = TestHelper.verify(() -> {
+      ExternalView externalView = _gSetupTool.getClusterManagementTool().getResourceExternalView(CLUSTER_NAME, DATABASE);
+      Map <String, String> stateMap = externalView.getStateMap(DATABASE + "_0");
+      if (!stateMap.containsKey(PARTICIPANT_PREFIX + "_" + (_startPort + 1))) {
+        return false;
+      }
+      return stateMap.get(PARTICIPANT_PREFIX + "_" + (_startPort + 1)).equals("MASTER");
```

Comment:
These checks could be simplified to `"MASTER".equals(stateMap.get(PARTICIPANT_PREFIX + "_" + (_startPort + 1)));`

Reply:
I don't see much difference. Anyway I change it as you suggested.